dist/cargo_test.sh

```diff
@@ -23,52 +24,20 @@ declare -A executors
 executors["cargo"]="execute_native"
 
 function run_tests() {
+  curl -LsSf https://get.nexte.st/latest/linux | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
   set -x
   export CARGO_TARGET_DIR="$PWD/target"
 
-  if [[ $(type -f kcov) ]]; then
-    export HAS_KOV="${HAS_KOV:-1}"
-    rm -rf "${CARGO_TARGET_DIR}"/cov
-  fi
+  ARTIFACT_DIR="${ARTIFACT_DIR:-${PWD}}"
 
   has_failed=false
   for directory in ${!cargo_test_flags[*]}; do
-    if [[ "${HAS_KOV}" -eq "1" ]]; then
-      # we want to prevent completely untested functions to be stripped
-      export RUSTFLAGS='-C link-dead-code'
-    fi
-
-    if ! /usr/bin/env bash -c "\
-        set -xe
-        cd ${directory}
-        cargo test --no-run
-        mapfile -t tests < <(
-          cargo test --no-run --message-format=json ${cargo_test_flags[${directory}]} | \
-            jq -r 'select(.profile.test == true) | .executable'
-        )
-        for test in \${tests[@]}; do
-          if [[ \"${HAS_KOV}\" -eq \"1\" ]]; then
-            kcov \
-              --exclude-pattern=$HOME/.cargo \
-              --verify \
-              ${CARGO_TARGET_DIR}/cov \
-              \$test
-          else
-            \$test
-          fi
-        done
-      "; then
+    if ! cargo nextest run ${cargo_test_flags[$directory]} \ --package "${directory}" --profile ci ; then
```

Comment:
Addressing the https://github.com/openshift/cincinnati/pull/936#issuecomment-2334239524 comment. 

I am not sure what the `\` slash's purpose is here.

In the logs we get (note that the `--package` flag is in apostrophes):
```
$ grep -E "cargo nextest run" build-log.txt 
+ cargo nextest run --features test-net,test-net-private ' --package' quay --profile ci
+ cargo nextest run ' --package' prometheus-query --profile ci
+ cargo nextest run ' --package' policy-engine --profile ci
+ cargo nextest run --features test-net,test-net-private ' --package' graph-builder --profile ci
+ cargo nextest run --features test-net,test-net-private ' --package' cincinnati --profile ci
+ cargo nextest run ' --package' commons --profile ci
```

Locally running the PR changes:
```
[dhurta@fedora cincinnati]$ grep -Ei "Summary" PR_log.log 
     Summary [   0.000s] 0 tests run: 0 passed, 130 skipped
     Summary [   0.051s] 10 tests run: 3 passed, 7 failed, 132 skipped
     Summary [   0.000s] 0 tests run: 0 passed, 142 skipped
     Summary [   0.000s] 0 tests run: 0 passed, 130 skipped
     Summary [   0.000s] 0 tests run: 0 passed, 130 skipped
     Summary [  66.577s] 4 tests run: 3 passed, 1 failed, 138 skipped
```

By removing the slash from the script we get more tests running:
```
$ grep -Ei "Summary" PR_with_change_log.log 
     Summary [   0.003s] 1 test run: 1 passed, 0 skipped
     Summary [  67.315s] 73 tests run: 65 passed, 8 failed, 3 skipped
     Summary [   0.012s] 8 tests run: 8 passed, 0 skipped
     Summary [   0.057s] 14 tests run: 14 passed, 0 skipped
     Summary [   0.006s] 6 tests run: 6 passed, 0 skipped
     Summary [   0.966s] 2 tests run: 2 passed, 0 skipped
```

Reply:
I understand that it's an escape character; however I am not sure why

Reply:
Good catch!

> I am not sure what the \ slash's purpose is here.

Just a leftover of multiline command; I collapsed the line when it shortened but the unfortunate `\` stayed there, resulting in this funny interaction where whatever flag parser is inside does not recognize `' --flag'` as a flag :D